src/components/plots/rectanglePlot.ts

```diff
@@ -0,0 +1,110 @@
+///<reference path="../../reference.ts" />
+
+module Plottable {
+export module Plot {
+  export class Rectangle extends AbstractXYPlot<any, any> {
+    private _colorScale: Scale.AbstractScale<any, string>;
+
+    /**
+     * Constructs a RectanglePlot.
+     *
+     * A RectanglePlot consists of a bunch of rectangles. The user is required to
+     * project the top left corner of each rectangle (x1, y1) and the bottom right
+     * corner of each rectangle (x2, y2)
+     *
+     * @constructor
+     * @param {Scale.Ordinal} xScale The x scale to use.
+     * @param {Scale.Ordinal} yScale The y scale to use.
+     */
+    constructor(xScale: Scale.AbstractScale<any, any>, yScale: Scale.AbstractScale<any, any>,
+        colorScale: Scale.AbstractScale<any, string>) {
```

Comment:
I don't like having the colorScale in the constructor.  We are planning to remove these args from the constructor anyways so...